src/test/regress/sql/alter_table_set_access_method_view.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SCHEMA view_acceess_method_test;
```

Comment:
Typo in the word `access`. Check other occurrences of this typo in the same file.

Reply:
Fixed